specification/operationalinsights/data-plane/readme.md

```diff
@@ -25,7 +25,7 @@ These are the global settings for the OperationalInsightsData API.
 
 ``` yaml
 title: OperationalInsightsDataClient
-description: Operational Insights Data Client
+description: Log Analytics Data Plane Client
```

Comment:
Reflecting branding, which has been different for some time (keeping title due to existing packages dependent on it; will override on a per package basis for new releases.